deploy/docker/deploy_buttonmen_site

```diff
@@ -26,7 +26,8 @@ LOCALHOST_IPV4 = '127.0.0.1'
 def get_subprocess_output(cmdargs):
   return subprocess.check_output(cmdargs).decode()
 
-REPO_MATCH = re.compile('^origin\s+git@github.com:([\w-]+)/buttonmen.git \(fetch\)$')
+REPO_MATCH_SSH = re.compile('^origin\s+git@github.com:([\w-]+)/buttonmen.git \(fetch\)$')
+REPO_MATCH_HTTPS = re.compile('^origin\s+https://github.com/([\w-]+)/buttonmen.git \(fetch\)$')
```

Comment:
Can you make these changes to `get_buttonmen_site_ipaddr` and `teardown_buttonmen_site` as well?  You won't need them, but i've been keeping them in sync, in lieu of having had ten seconds to do anything halfway intelligent with modules.